qualcoder/GUI/ui_dialog_codes.py

```diff
@@ -62,7 +62,10 @@ def setupUi(self, Dialog_codes):
         self.splitter = QtWidgets.QSplitter(Dialog_codes)
         self.splitter.setOrientation(QtCore.Qt.Horizontal)
         self.splitter.setObjectName("splitter")
-        self.treeWidget = QtWidgets.QTreeWidget(self.splitter)
+        self.leftsplitter = QtWidgets.QSplitter(self.splitter)
```

Comment:
OK these changes need to be made to the UI file or they will be overridden, if the ui is changed in qtdesigner.